app.js

```diff
@@ -2,20 +2,27 @@ const express = require("express");
 const {
   handleCustomErrors,
   handlePSQL400s,
-  handle500sErrors
+  handle500sErrors,
 } = require("./controllers/errorHandlingControllers");
 const app = express();
-const { getCategories, getReviews, getReviewById } = require("./controllers/gamesController");
+const {
+  getCategories,
+  getReviews,
+  getReviewById,
+  getCommentsOfReview,
+} = require("./controllers/gamesController");
 
 app.get("/api/categories", getCategories);
 
 app.get(`/api/reviews`, getReviews);
 
-app.get(`/api/reviews/:review_id`, getReviewById)
+app.get(`/api/reviews/:review_id`, getReviewById);
+
+app.get(`/api/reviews/:review_id/comments`, getCommentsOfReview);
 
-app.use('*', (request, response, next) => {
-    response.status(404).send({ message: '404: Path not found!'})
-})
+app.use("*", (request, response, next) => {
+  response.status(404).send({ message: "404: Path not found!" });
```

Comment:
might be tempted to extract this out into your error handlers, but its totes fine